docker/kayobe/Dockerfile

```diff
@@ -22,13 +26,21 @@ ENV container docker
 # VOLUME [ "/sys/fs/cgroup" ]
 
 # CMD ["/usr/sbin/init"]
-
+# Note on CentOS 8 rsync-3.1.3-14.el8 install:
+# Workaround rsync: --sparse-block=1024: unknown option
+# Hopepfully they have fixed this issue before this old
+# package is removed.
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=2043753
+ARG BASE_IMAGE="quay.io/centos/centos:stream8"
 RUN dnf install epel-release -y && \
     dnf update -y --nobest && \
-    dnf install -y gcc git vim python3-pyyaml python3-virtualenv \
-        libffi-devel sudo which openssh-server e2fsprogs rsync \
+    dnf install -y gcc git vim python3-pyyaml \
+        libffi-devel sudo which openssh-server e2fsprogs \
         diffstat diffutils debootstrap procps-ng gdisk util-linux \
         dosfstools lvm2 kpartx systemd-udev bash-completion && \
+    if [ "$BASE_IMAGE" == "rockylinux:9" ] ; then \
```

Comment:
This is a bit specific to the image name & tag, which we might change using an argument.

How about sourcing /etc/os-release and checking the version?

Reply:
Done